modules/distr-tracing-tempo-install-tempomonolithic-web-console.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempomonolithic-web-console_{context}"]
+= Installing a TempoMonolithic instance by using the web console
+
+:FeatureName: The TempoMonolithic instance
+include::snippets/technology-preview.adoc[]
+
+You can install a TempoMonolithic instance from the *Administrator* view of the web console.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a cluster administrator with the `cluster-admin` role.
+
+* For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Create a project of your choice for the *TempoMonolithic* instance that you will create in a subsequent step: go to *Home* -> *Projects* -> *Create Project*.
+
+. Decide which type of supported storage you will use for storing traces: in-memory storage, a Persistent Volume, or object storage.
++
+[IMPORTANT]
+====
+Object storage is not included with the {TempoShortName} and requires setting up an object store by a supported provider: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], or link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+Additionally, opting for object storage requires creating a secret for your object storage bucket in the project that you created for the *TempoMonolithic* instance: go to *Workloads* -> *Secrets* -> *Create* -> *From YAML*.
```

Comment:
```suggestion
Additionally, opting for object storage requires creating a secret for your object storage bucket in the project that you created for the *TempoMonolithic* instance. To create a secret, click *Workloads* -> *Secrets* -> *Create* -> *From YAML*.
```

Reply:
Thank you. The intended audience knows what to do. The menu navigation is just a tip for their convenience.

Reply:
This is how I reworked it:

```
Additionally, opting for object storage requires creating a secret for your object storage bucket in the project that you created for the *TempoMonolithic* instance.
+
[TIP]
====
You can do this in *Workloads* -> *Secrets* -> *Create* -> *From YAML*.
====
```